src/index.css

```diff
@@ -1,13 +1,37 @@
 body {
```

Comment:
- [ ] [OPTIONAL] Your project is looking great! Just make sure to set the color, or height, of body so a white margin doesn't show up under the page and ruins your beautiful design:
![Screenshot_2020-10-01 React App](https://user-images.githubusercontent.com/16193212/94834525-da9cc280-03e6-11eb-94ed-684867715236.png)